### DISCUSSION TITLE
Trashed items ALIAS confusion

### DISCUSSION THREAD

**Discussion by kitepascal** [joomla/joomla-cms#37013] [2. Cold Features]
Since always, the ALIAS conflict when saving items is confusing people. The warning text is more clear now but it could be even better.

Save failed with the following error: The alias ALIAS is already being used by ALIAS menu item in the Main Menu (en-EN) menu (**remember it may be a trashed item**).

Maybe an extension of that warning would be useful?
A direct link to that trashed item or a button [ Delete that trash and continue saving ].

Because most Joomla newbies don't even know how to access the trash bin.

See https://forum.joomla.de/thread/15322-spezifisches-men%C3%BC-alias-funktioniert-nicht/ (German).

**Comment by chmst** [comment]
As the message says ... MAYBE is the element trashed. Maybe not .. 
But a link to the item with the existing alias could really be useful.

**Comment by brianteeman** [comment]
please take a look at #37017

**Comment by kitepascal** [comment]
Thanks a lot! 🙏

**Comment by Hackwar** [comment]
Since this has been fixed, I'm closing this discussion.
